cmd/ww/server.go

```diff
@@ -287,6 +320,8 @@ func server(args ...string) {
 	stunservers := set.String("stun", "stun:relay.webwormhole.io", "list of STUN server addresses to tell clients to use")
 	set.StringVar(&turnServer, "turn", "", "TURN server to use for relaying")
 	set.StringVar(&turnSecret, "turn-secret", "", "secret for HMAC-based authentication in TURN server")
+	set.StringVar(&customCertFlag, "custom-cert", "", "Custom TLS certificate")
```

Comment:
let's drop the 'custom' prefix, and keep it `cert` and `key`. if they're provided we use them, if not we default to let's encrypt.

description can be "certificate for HTTPS, leave empty to use Let's Encrypt"